charts/federation-v2/charts/webhook/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
I would be fine to fold it into the existing chart, but it's still currently factored as a 'controller-manager' subchart. Seems like this would probably be better to do in a follow-up to me, wdyt?

Reply:
I'm fine with that so long as it gets done before the next release.

Reply:
I've now moved the chart to be within the `controller-manager` subchart. I think in the future we should just have one `kubefed` chart.